scripts/publish_prod.sh

```diff
@@ -23,7 +23,7 @@ CURRENT_VERSION=$(node -pe "require('./package.json').version")
 if [ -z "$1" ]; then
     echo "Must specify a desired version number"
     exit 1
-elif [[ ! $1 =~ [0-9]+\.[0-9]+\.[0-9]+ ]]; then
+elif [[ ! $1 =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
NOTE `3.2.222222` is allowed, should we stop this? maybe only allow each num 0-9?